### ISSUE TITLE
Redesign Gaze Cursor, Add loader state version

### ISSUE THREAD

**Issue by mattatgit** [vizorvr/vizor#1227]
1. gazeclick cursor is very heavy (thick) and distracting
2. We need some kind of loading animation for the case of loading files from trigger (i.e. the player is already loaded, we are just loading another file into it). This can't use the player loading linear load indicator, as this would just be weird.
* solution: design another state for the gaze click cursor to indicate file loading (as if there are triggers being used, the gaze click cursor must be present

**Comment by fthr** [comment]
I see two options for making the transition between two files smotoher.

1 Activate a "load file on trigger".  
2 Loading is indicated inside that scene (for example in the gaze cursor as you say), 
3 When loaded jump into new scene.

or

1. Activate a "load file on trigger"
2. Jump into a "loading sphere" where there is a VR progress bar. This could be a darkly colored skysphere so not to strain the eyes.
3. Jump into the loaded scene.

**Comment by mattatgit** [comment]
yeah. I'd say ideally we would just go from one to another, with transition, following simple, nice load animation based in the cursor (because then you can look away around the current scene if you want, but still be aware of progress)
